lib/collection/src/operations/types.rs

```diff
@@ -1019,6 +1019,18 @@ impl CollectionError {
             Self::StrictMode { .. } => false,
         }
     }
+
+    pub fn is_pre_condition_failed(&self) -> bool {
+        matches!(self, Self::PreConditionFailed { .. })
+    }
+
+    pub fn is_missing_point(&self) -> bool {
+        match self {
+            CollectionError::NotFound { what } => what.contains("No point with id"),
```

Comment:
Maybe use something more reliable?

Reply:
Opened an issue for that https://github.com/qdrant/qdrant/issues/5063